installer/templates/static/brunch/package.json

```diff
@@ -6,8 +6,8 @@
     "watch": "brunch watch --stdin"
   },
   "dependencies": {
-    "phoenix": "file:<%= brunch_deps_prefix %><%= phoenix_path %>"<%= if html do %>,
-    "phoenix_html": "file:<%= brunch_deps_prefix %>deps/phoenix_html"<% end %>
+    "phoenix": "~<%= phoenix_version %>"<%= if html do %>,
+    "phoenix_html": "~<%= phoenix_version %>"<% end %>
```

Comment:
@chrismccord Ah, right. I'll correct that.
@josevalim So publish `phoenix` to npm, but don't depend on it? Since they're already using npm to manage dependencies (`brunch` comes in via npm, correct?) and it's a planned part of the new release workflow, I don't see a downside.